### DISCUSSION TITLE
[Tests] Add model specs for Curriculum and CurriculumStory

### DISCUSSION THREAD

**Discussion by slaloggia** [Terrastories/terrastories#740] [Temporarily Archived Tickets]
New models need tests!
Optional: add factories for each

**Comment by mirandawang** [comment]
@rgenchev mentioned wanting to write tests for CurriculumStory model, so I'm assigning this ticket here :)

**Comment by rgenchev** [comment]
Thanks, @mirandawang! I'm on it! :)

**Comment by rgenchev** [comment]
@mirandawang, I decided to split this issue into two in order to have single-purpose PRs. Here are the two new issues: https://github.com/Terrastories/terrastories/issues/547 and https://github.com/Terrastories/terrastories/issues/548 . I think that this one can be closed.
